src/content/en/fundamentals/engage-and-retain/push-notifications/common-issues-and-reporting-bugs.md

```diff
@@ -0,0 +1,271 @@
+
+1. **Sending a Message:** Make sure that sending messages is successful.
+   You should be getting a 201 HTTP code. If you aren't :
+    1. **Check for Authorization Errors:** If you receive an authorization
+       error message see the
+       [Authorization Issues section](#authorization_issues).
+    1. **Other API Errors:** If you receive a non-201 status code response,
+       see the [HTTP Status Codes section](#http_status_codes) for
+       guidance on the cause of the issue.
+1. **Receiving a Message**: If you're able to send a message successfully,
+   but the message is not received on the browser:
+    1. **Check for Encryption Issues:** See the [Payload Encryption
+       Issue Section](#payload_encryption_issue).
+    1. **Check for Connection Issues:** If the problem is on Chrome, it
+       may be a connection, see [Connection Issues section](#connection_issue)
+       for more info.
+
+If you aren't able to send and receive a push message and the relevant sections
+in this doc aren't helping debug / fix the problem then you may have found a
+bug. In this case, refer to the [Raising Bug Reports](#raising_bug_reports)
+section to file a good bug report with all the necessary information to expedite
+the bug fixing process.
+
+One thing I'd like to call out before we start is that **Firefox and the
+Mozilla AutoPush Service have great errors messages.** If you get stuck and not
+sure what the problem is, be sure to test in Firefox and see if you get a
+more helpful error message.
+
+## Authorization Issues
+
+Authorization issues are one of the most common issues developers hit when
+starting out with WebPush. This is normally a problem with configuration of a
+sites [Application Server Keys / VAPID
+Keys](https://tools.ietf.org/html/draft-ietf-webpush-vapid-02).
+
+The easiest way to support push in both Firefox and Chrome is to supply an
+`applicationServerKey` in the `subscribe()` call. The down side is that
+any discrepancy between your front end and servers keys will result in an
+authorization error.
+
+### On Chrome + FCM
+
+For Chrome, which uses FCM as a push service, you'll receive an
+`UnauthorizedRegistration` response from FCM for a range of different
+errors, all involving the application server keys / VAPID keys.
+
+You'll receive an `UnauthorizedRegistration` error in any of the following
+situations:
+
+* If you fail to define an `Authorization` header in the request to FCM.
+* Your application key used to subscribe the user doesn't match the key to sign
+  the Authorization header.
+* The expiration is invalid in your JWT, i.e. the expiration exceeds 24 hours or
+  the JWT has expired.
+* The JWT is malformed or has invalid values.
+
+The full error response looks like this:
+
+```
+<HTML>\n<HEAD>\n<TITLE>UnauthorizedRegistration</TITLE>\n</HEAD>\n<BODY BGCOLOR="#FFFFFF" TEXT="#000000">\n<H1>UnauthorizedRegistration</H1>\n<H2>Error 400</H2>\n</BODY>\n</HTML>\n
+```
```

Comment:
This is perfectly valid markdown and either shouldn't be support at all OR the warnings should be fixed to support this. The entire point of the triple back tick is to avoid ambiguity of the 4 spaces.